src/videojs-contrib-hls.js

```diff
@@ -547,7 +547,7 @@ class HlsHandler extends Component {
       this.ignoreNextSeekingEvent_ = true;
     });
 
-    this.setupQualityLevels_();
+    this.tech_.ready(this.setupQualityLevels_.bind(this));
```

Comment:
er, wait, never mind. Ignore me. However, we shouldn't use `.bind` because it's slow and doesn't run everywhere where we want to run.
You should be able to use an arrow function instead.